framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/Domain/Repositories/RepositoryExtensions.cs

```diff
@@ -47,6 +47,33 @@ public static class RepositoryExtensions
         }
     }
 
+
+    public static async Task EnsureExistsAsync<TEntity, TKey>(
+           this IRepository<TEntity, TKey> repository,
+           TKey id,
+           CancellationToken cancellationToken = default
+       )
+           where TEntity : class, IEntity<TKey>
+    {
+        if (await repository.AnyAsync(x => x.Id.Equals(id), cancellationToken))
```

Comment:
do you mean 
```suggestion
        if (!await repository.AnyAsync(x => x.Id.Equals(id), cancellationToken))
```
?
if, also need change below method

Reply:
Yes. I forgot that :)